lib/ansible/constants.py

```diff
@@ -59,6 +59,15 @@ def __getitem__(self, y):
         return self._value[y]
 
 
+def _add_builtin_fqcn(names):
```

Comment:
function is fine, but this might not be best location, as constants.py has a heavy cost on import for those that already dont use it.

Reply:
I first thought about lib/ansible/utils/collection_loader/__init__.py, but that comes with its own import tree. How about lib/ansible/utils/fqcn.py (a new file)? Or lib/ansible/utils/helpers.py?

Reply:
either works for me, you can even cache responses with a module level var